### ISSUE TITLE
Ambiguity in Polynomial Regression Notes

### ISSUE THREAD

**Issue by NicolasRiley** [rtealwitter/datamining2025#2]
While reading the notes, I found two areas that seemed ambiguous to me.

The first comes from a line in the first paragraph which says "We train the model on the training set, and measure the performance on both the training set and the validation set." Because I was a bit confused on this, I looked it up and found that the validation set plays a much larger role in Regularization (and the training set is used to track overfitting). The statement implies these two sets are equally integral to regularization when they are not.

<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/2824d197-aa9d-49bd-8fac-981083af600b" />

The second was in the histogram of weights. I'm not sure if it's just a display issue for my laptop but the colors of the bars don't align that well with the legend. The 2nd bar from the left (assuming it is degree 2) appears darker than the bars for degree 10. It is also hard to differentiate between degree 1 and 2 bars.

<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/95ea721d-c5e3-4e39-bbe7-05cfcd10e3d7" />

**Comment by rtealwitter** [comment]
Thanks for noting these! I don't think the following sentence is ambiguous (it's factual):

> We train the model on the training set, and measure the performance on both the training set and the validation set.

Your point is that there's a reason we use the training set, which is also covered in the rest of the section.

In terms of the plot, it's not ideal coloring, but I also don't think it's ambiguous. The point is that the degree 10 weights have huge magnitude, and the smaller degree weights do not
